src/core/server/integration_tests/ci_checks/saved_objects/check_registered_types.test.ts

```diff
@@ -520,20 +520,20 @@ describe('checking migration metadata changes on all registered SO types', () =>
         "dashboard|10.3.0: 663cec8172649999f4d44c505d3c878cb7403014",
         "dashboard|10.2.0: 54c6d0a873cb020539682b083141ec6a1f987dd7",
         "dashboard|10.1.0: 3c7162cbfafabddf7400ec2f034a0f568069862c",
-        "dashboard|8.4.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|8.0.1: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|8.9.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|8.6.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|7.16.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|7.15.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|8.5.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|8.3.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|8.1.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|8.0.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|7.17.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|7.13.1: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|8.7.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
-        "dashboard|8.2.0: 9c2a48b37df87d7d724105d2d4eb0ca0e0b7d95d",
+        "dashboard|8.4.0: 062db9fffc526bc492f39fd0cef8ab84e6f6d3fb",
```

Comment:
Any idea what's going on with these? Auto-generated I assume?

Reply:
Auto-generated, yup 🤷 Saved object stuff since I added `autoApplySelections` to the dashboard SO